.buildkite/node-pipeline.yml

```diff
@@ -1,78 +1,80 @@
 steps:
 
-  - label:  ':docker: Build node maze runner image'
-    key: "node-maze-runner-image"
-    timeout_in_minutes: 20
-    plugins:
-      - artifacts#v1.5.0:
-          download: min_packages.tar
-          build: ${BUILDKITE_TRIGGERED_FROM_BUILD_ID}
-      - docker-compose#v3.9.0:
-          build:
-            - node-maze-runner
-          image-repository: 855461928731.dkr.ecr.us-west-1.amazonaws.com/js
-          cache-from:
-            - node-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-node-${BRANCH_NAME}
-      - docker-compose#v3.9.0:
-          push:
-            - node-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-node-${BRANCH_NAME}
+  - group: "Node Tests"
+    steps:
+      - label:  ":docker: Build node maze runner image"
+        key: "node-maze-runner-image"
+        timeout_in_minutes: 20
+        plugins:
+          - artifacts#v1.5.0:
+              download: min_packages.tar
+              build: ${BUILDKITE_TRIGGERED_FROM_BUILD_ID}
+          - docker-compose#v3.9.0:
+              build:
+                - node-maze-runner
+              image-repository: 855461928731.dkr.ecr.us-west-1.amazonaws.com/js
+              cache-from:
+                - node-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-node-${BRANCH_NAME}
+          - docker-compose#v3.9.0:
+              push:
+                - node-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-node-${BRANCH_NAME}
 
-  - label: ':node: Node 4'
-    depends_on: "node-maze-runner-image"
-    timeout_in_minutes: 30
-    plugins:
-      docker-compose#v3.9.0:
-        run: node-maze-runner
-        use-aliases: true
-    env:
-      NODE_VERSION: "4"
+      - label: ":node: Node 4"
```

Comment:
Not related to this PR but this looks like it could be easily matrix-ed

Reply:
Agreed, should well well here, have raised PLAT-9115.